docs/guide/service/annotations.md

```diff
@@ -49,7 +49,8 @@
 | [service.beta.kubernetes.io/aws-load-balancer-alpn-policy](#alpn-policy)                         | string                  |                           |                                                        |
 | [service.beta.kubernetes.io/aws-load-balancer-target-node-labels](#target-node-labels)           | stringMap               |                           |                                                        |
 | [service.beta.kubernetes.io/aws-load-balancer-attributes](#load-balancer-attributes)             | stringMap               |                           |                                                        |
-| [service.beta.kubernetes.io/aws-load-balancer-security-groups](#security-groups)                 | stringList              |                           |                                                        | 
+| [service.beta.kubernetes.io/aws-load-balancer-security-groups](#security-groups)                 | stringList              |                           |                                                        |
+| [service.beta.kubernetes.io/aws-load-balancer-extra-security-groups](##security-groups)          | stringList              |                           |                                                        |
```

Comment:
Should this have a unique anchor tag? Like `#extra-security-groups`

Reply:
got it, adding